README.md

```diff
@@ -120,3 +116,32 @@ Tweet at me! [@TheRealPomax](http://twitter.com/TheRealPomax) or [@TheRealPomax@
 ## And if I just want to use this?
 
 This code is [MIT licensed](https://raw.githubusercontent.com/Pomax/Font.js/master/LICENSE), do whatever you want with it.
+
+## NodeJS
+
+New to Font.js is the possibility to load the information of a font file located on your server, via NodeJS. Be sure to have the latest [NodeJS](https://nodejs.org/en/download/current/) installed, and run:
+```bash
+$ npm install
+```
+Create an example `font-node.js` file:
+```js
+import FontNode from './FontNode.js';
+
+const file = process.argv[2];
```

Comment:
Maybe for simplicity's sake, leave out the argument, and use a hardcoded file in the example (`const file = "./fonts/SourceCodePro-Regular.ttf"`). The user can then decide how they want to grab the file.